roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
Likewise, I'm not sure that origin + atomic-enterprise is a valid option.

I think we could squash this down to one variable with 'origin' 'online' 'atomic-enterprise' 'openshift' being valid options. origin gets the origin package/services. online I'm uncertain of, the last two getting atomic-openshift package/services.

product_type never really made it's way into use, so if we want to squash this into one knob we should do it now.